src/identity.js

```diff
@@ -416,6 +417,37 @@ export class Identity extends EventEmitter {
         return parseInt(safariVersion[1], 10) >= 12;
     }
 
+    /**
+     * Log current using version of sdk
+     * @throws {SDKError} - If log method is not provided
+     * @return {void}
+     */
+    printVersion() {
+        if (!this.log) {
+            throw new SDKError('You have to provide log method in constructor');
+        }
+
+        this.log(`Schibsted account SDK for browsers version: ${version}`);
+    }
+
+    /**
+     * Log used settings
+     * @throws {SDKError} - If log method is not provided
+     * @return {void}
+     */
+    printSettings() {
```

Comment:
Could we have one function that will print version and settings in the same time?